### PR TITLE
run under convox/docker

### PR DESCRIPTION
**Pull request by macool**
```
convox start
```

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;gYPar1oC&#x2F;650-qae-support-server-maintenance"> QAE Support: Server Maintenance</a>

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM phusion/baseimage:latest
```
Comment (matthewford) [inline Dockerfile]: Image is too large, can we use the gliderlabs/alpine image

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    - 3000:3000
+  volumes:
+    - .:/app
+  links:
+    - db
+    - redis
+worker:
+  <<: *web
+  ports: []
+  command: bundle exec sidekiq -L ./log/worker.log
```
Comment (matthewford) [inline docker-compose.yml]: should log to stdout

**Comment by matthewford** [comment]
Can we change the docker_compose file to `docker_compose.yml.local` which you can specify on start. 

This is because this is the file convox uses on deployment.

**Comment by macool** [comment]
Do not merge please. I'm still failing with

`web    | /usr/lib/ruby/gems/2.2.0/gems/bundler-1.11.2/lib/bundler/runtime.rb:80:in `rescue in block (2 levels) in require': There was an error while trying to load the gem 'skylight'. (Bundler::GemRequireError)`

**Comment by matthewford** [comment]
Remove skylight, it uses rust so could be missing a dependency.

sent from my phone.

**Comment by macool** [comment]
@matthewford I simplified circle file by adding `sql` structure dump

File: config/application.rb [hunk 3]
```diff
@@ -33,6 +33,7 @@ class Application < Rails::Application
     end
 
     config.cache_store = :memory_store
+    config.active_record.schema_format = :sql
```
Comment (matthewford) [inline config/application.rb]: @macool don't we need to update the schema then?

Comment (matthewford) [inline config/application.rb]: Ignore me, I didn;t see the file as github hid it.

**Pull request merged**